MapPointMapper.xcodeproj/project.pbxproj

```diff
@@ -195,6 +195,7 @@
 		547424DC1A1C1CE800A8478C /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
+				LastSwiftUpdateCheck = 0700;
```

Comment:
Added by Xcode 7 automatically. Opening a project with Swift in it adds that key